packages/admin-panel-server/src/app/createApp.ts

```diff
@@ -49,6 +53,8 @@ export function createApp() {
     .useSessionModel(AdminPanelSessionModel)
     .verifyLogin(hasTupaiaAdminPanelAccess)
     .get('/v1/hierarchy/:hierarchyName/:entityCode', handleWith(FetchHierarchyEntitiesRoute))
+    .get('/v1/fetchReportPreviewData', handleWith(FetchReportPreviewData))
+    .post('/v1/saveDashboardVisualisation', handleWith(SaveDashboardVisualisation))
```

Comment:
Since viz-builder endpoints live in admin-panel-server now, I think we have to name our endpoints a bit more specific (eg: cannot name /save anymore). I have a thought that we should have all the endpoints for viz-builder to start with `vizBuilder`, ie:
- `/v1/vizBuilder/fetchPreviewData`
- `/v1/vizBuilder/save`

That way we don't have to be too specific with our endpoints naming anymore. Also, if we want to check for permission for all viz-builder endpoints, we can easily add a middleware for endpoints that match `vizBuilder`?